client/app/scripts/superdesk-vocabularies/views/vocabulary-config-modal.html

```diff
@@ -0,0 +1,36 @@
+<form name="editForm">
+  <div sd-modal data-model="vocabulary" class="add-dictionary-popup">
+    <div class="modal-header">
+      <a href="" class="close" ng-click="cancel()"><i class="icon-close-small"></i></a>
+      <h3 translate>Edit Vocabulary: {{vocabulary._id}}</h3>
+    </div>
+
+    <div class="modal-body">
+      <div class="upload-progress" ng-if="progress.width">
+        <div class="bar" style="width: {{ progress.width }}%;"></div>
+      </div>
+
+      <ng-form name="vocabularyForm" class="vocabularyForm">
+
+        <fieldset>
+            <div ng-repeat="(key, value) in model" class="field vocabularyField">
+              <label translate>{{key}}</label>
+            </div>
+          <div ng-repeat="item in vocabulary.items track by $index">
+            <div ng-repeat="(key, value) in model" class="field vocabularyField">
+              <input type="text" ng-model="item[key]">
+            </div>
+            <button id="del-btn" class="btn del" ng-click="removeItem(item)" translate>x</button>
```

Comment:
it was a joke*, i did not changed it during development when label was `delete`

*https://www.reddit.com/r/Games/comments/1f3ron/why_is_o_yes_an_x_no_in_japanese_playstation/